packages/schema-blocks/src/core/blocks/BlockDefinition.tsx

```diff
@@ -51,8 +51,14 @@ export default class BlockDefinition extends Definition {
 
 		if ( sidebarElements.length > 0 ) {
 			// Need to add `children` on the `props` as well, because of the type definition of `InspectorControls.Props`.
-			const sidebar = createElement( PanelBody, { key: "sidebarPanelBody", children: sidebarElements }, sidebarElements );
-			const sidebarContainer = createElement( InspectorControls, { key: "sidebar", children: [ sidebar ] }, [ sidebar ] );
+			const sidebar = createElement( PanelBody, {
+				key: "sidebarPanelBody",
+				children: sidebarElements,
+			}, sidebarElements );
+			const sidebarContainer = createElement( InspectorControls, {
+				key: "sidebar",
```

Comment:
sidebar vind ik ook nog eng generiek; mss block name erbij? e.g.`job-postingsidebar`
```suggestion
				key: this.options.name + "Sidebar",
```